### PR TITLE
Update Localizable.xcstrings

### PR DESCRIPTION
**Pull request by Svec-Tomas**
Improvement of the Slovak translation

### CODE REVIEW

File: Easydict/App/Localizable.xcstrings [hunk 1]
```diff
@@ -6325,7 +6325,7 @@
         "sk" : {
           "stringUnit" : {
             "state" : "translated",
-            "value" : "Access Key ID"
```
Comment (Jerry23011) [inline Easydict/App/Localizable.xcstrings:6328]: This doesn't need to be translated.

File: Easydict/App/Localizable.xcstrings [hunk 2]
```diff
@@ -6359,7 +6359,7 @@
         "sk" : {
           "stringUnit" : {
             "state" : "translated",
-            "value" : "Secret Access Key"
```
Comment (Jerry23011) [inline Easydict/App/Localizable.xcstrings:6362]: This doesn't need to be translated.

**Review by Jerry23011 — CHANGES_REQUESTED** [review]

File: Easydict/App/Localizable.xcstrings (hunk 2 above)
Comment (Svec-Tomas) [reply Easydict/App/Localizable.xcstrings]: Do I have to do something now? Or is it OK now?

File: Easydict/App/Localizable.xcstrings [hunk 3]
```diff
@@ -4840,7 +4840,7 @@
         }
       }
     },
-    "request_screen_capture_access_description" : {
+    "request_screen_capture__description" : {
```
Comment (Jerry23011) [inline Easydict/App/Localizable.xcstrings:4843]: You seem to have changed the original string.

File: Easydict/App/Localizable.xcstrings [hunk 4]
```diff
@@ -5012,12 +5012,12 @@
         }
       }
     },
-    "service.configuration.ali.access_key_id.title" : {
+    "service.configuration.ali._key_id.title" : {
```
Comment (Jerry23011) [inline Easydict/App/Localizable.xcstrings:5015]: You seem to have changed the original string.

File: Easydict/App/Localizable.xcstrings [hunk 5]
```diff
@@ -5012,12 +5012,12 @@
...
       "localizations" : {
         "en" : {
           "stringUnit" : {
             "state" : "translated",
-            "value" : "AccessKey ID"
+            "value" : "Key ID"
```
Comment (Jerry23011) [inline Easydict/App/Localizable.xcstrings:5020]: Please revert the change here

**Review by Jerry23011 — CHANGES_REQUESTED** [review]

File: Easydict/App/Localizable.xcstrings (hunk 2 above)
Comment (Jerry23011) [reply Easydict/App/Localizable.xcstrings]: @Svec-Tomas Please check the latest review, thanks.

File: Easydict/App/Localizable.xcstrings (hunk 3 above)
Comment (Svec-Tomas) [reply Easydict/App/Localizable.xcstrings:4843]: I don't know what to do now. I am not very familiar with Github. I just wanted to update the translation. Can you please fix it or should I make a new PR as I have no idea what to do now?

**Review by Jerry23011 — APPROVED** [review]
Thanks for contributing @Svec-Tomas

File: Easydict/App/Localizable.xcstrings (hunk 3 above)
Comment (Svec-Tomas) [reply Easydict/App/Localizable.xcstrings]: Thanks a lot Jerry for your help!

I just edited the file here on Github.

Xcode is meant to be for developers. Xcode is really unnecessary in my opinion as it is too big and really complicated for someone who just wants to translate something. There are other lightweight and small apps, that can be used for translating apps:

[Loca Studio](https://apps.apple.com/sk/app/loca-studio/id1465684707?l=sk&mt=12) for example. I believe lot of other users who can contribute with translation including myself finds Xcode really complicated and it is much easier to just download a file for translation and import it to for example to the free Loca Studio application. Maybe you could try it and update the guide? I think a simple way of translating could make more people to help to translate Easydict to their languages.

**Comment by tisfeng** [comment]
@Svec-Tomas Thanks for your PR.

We usually develop on the dev branch, which is ahead of the main branch. Currently, it has added some new localized texts. If you have time, please help us check the Slovak localization of the dev branch.

For the next PR, please develop and submit PR based on dev.

**Comment by Svec-Tomas** [comment]
Hi @tisfeng

I primarily use GitHub for the purpose of localizing applications into my native language. I truly value it when developers make the translation process as smooth as possible, for instance by providing support for other translation tools or offering detailed instructions. Unfortunately, I'm unfamiliar with the concept of a dev branch. As for my translation workflow, I avoid using Xcode due to its complexity. I've discovered more intuitive tools that are both lightweight and easy to use.

**Comment by Jerry23011** [comment]
>  Unfortunately, I'm unfamiliar with the concept of a dev branch

@Svec-Tomas This is not about what tools you use. When you download (or pull) xcstring from GitHub, please use the dev branch: https://github.com/tisfeng/Easydict/tree/dev

I understand that Xcode is heavy, and git can be complicated. You are doing an incredible job learning to PR on GitHub, feel free to use whatever tool you want. We are just here giving you suggestions to best align with our development goals.

**Comment by Svec-Tomas** [comment]
I have went through all translations and fixed some translations. Hopefully this time everything is OK. Thank you.
